synapse/lib/view.py

```diff
@@ -167,6 +167,17 @@ def getMergeRequest(self):
         if byts is not None:
             return s_msgpack.un(byts)
 
+    async def getMergeCount(self):
+        if self.info.get('quorum') is None:
+            mesg = f'View ({self.iden}) does not require quorum voting.'
+            raise s_exc.BadState(mesg=mesg)
+        count = 0
+        for view in list(self.core.views.values()):
+            await asyncio.sleep(0)
+            if view.parent == self and view.getMergeRequest() is not None:
+                count += 1
+        return count
```

Comment:
would we want to return the list of idens instead of a count?
With a list of idens we have a count AND the knowledge of who has a open MR.

Reply:
Yea, view idens aren't exactly secret knowledge, and with that we can still answer the main ask of "hey, there's outstanding merge requests to this view, maybe don't disable quorum?"./